docs/application/get-started/mobile.md

```diff
@@ -0,0 +1,38 @@
+# Get Started for Mobile
+
+## .NET Application
+
+When you create your first Tizen .NET application, you can select from the following options that uses C# as the programming language:
+
+-   [Mobile .NET application](../dotnet/get-started/mobile/first-app.md)
+
+    A mobile application is targeted to run on a smart phone. The applications can take advantage of a wide variety of functionalities available on smart phones.
+
+<a href="https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen" target="_blank">
+<img src="media/ic_docs_download.png"><Strong> Download Visual Studio Tools for Tizen</strong></a>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The Visual Studio Tools for Tizen is a collection of tools for developing Tizen .NET application.
+
+## Native Application
+
+When creating your first Tizen native application, you can select from the following options, which all use the C language:
+
+-   [Mobile native application](../native/get-started/mobile/first-app.md)
+
+    A mobile application is targeted to run on smart phones, and can take advantage of a wide variety of functionalities available on those devices.
+
+## Web Application
+
+When creating your first Tizen Web application, you can select from the following options, which all use the Web-native languages, such as HTML5, CSS, and JavaScript:
+
+-   [Mobile Web application](../web/get-started/mobile/first-app.md)
+
+    A mobile application is targeted to run on smart phones, and can
+    take advantage of a wide variety of functionalities available on
+    those devices.
+
+<a href="https://developer.tizen.org/development/tizen-studio/download" target="_blank">
```

Comment:
I changed the sentences. I think it is better to link download rather than to link overview page.
When a new window to download Tools is opened, developers can read the guides if they don't know.